### ISSUE TITLE
Muted role for every new text channel

### ISSUE THREAD

**Issue by ghost** [Cog-Creators/Red-DiscordBot#5111]
### Type of feature request

Cog, Other

### Description of the feature you're suggesting

in my opinion, the bot should add the Muted role with its permissions denied for each new channel created. In such a way as not to waste time for the admin of the server to add the role for each individual channel.

**Comment by Slurpcy** [comment]
Yes i would too agree on this one, as its too time consuming especially if u have many channels
